R/rules-line_break.R

```diff
@@ -66,18 +66,18 @@ set_line_break_after_opening_if_call_is_multi_line <-
   function(pd,
            except_token_after = NULL,
            except_text_before  = NULL) {
-  if (!is_function_call(pd)) return(pd)
-  npd <- nrow(pd)
-  is_multi_line <- any(pd$lag_newlines[seq2(3, npd - 1)] > 0)
-  if (!is_multi_line) return(pd)
-  exception_pos <- c(
-    which(pd$token %in% except_token_after),
-    ifelse(pd$child[[1]]$text[1] %in% except_text_before, 3L, NA)
-  )
-  pd$lag_newlines[3] <- 0L
-  pd$lag_newlines[setdiff(3, exception_pos)] <- 1L
-  pd
-}
+    if (!is_function_call(pd)) return(pd)
```

Comment:
Indention desired if `function()` is on new line?

Reply:
See line 66.

Reply:
Not really, but we can postpone. For now, maybe move `function` one line up?

Reply:
the problem is that the function name is so long that it does not all fit on one line...